packages/ui-patterns/Dialogs/ConfirmationModal.tsx

```diff
@@ -12,6 +11,7 @@ import {
   DialogTitle,
   cn,
 } from 'ui'
+import { Admonition } from './../admonition'
```

Comment:
Shouldn't the Admonition.tsx file name be capitalized?

Reply:
will sort it later